pallets/loans/src/mock.rs

```diff
@@ -246,6 +246,12 @@ impl ExtBuilder {
                 (CurrencyId::USDT, Ratio::from_percent(50)),
                 (CurrencyId::xDOT, Ratio::from_percent(50)),
             ],
+            reserve_factor: vec![
+                (CurrencyId::DOT, Ratio::from_percent(15)),
```

Comment:
will 15% be too high?

Reply:
15% comes from the USDT market in Filda, and can be adjusted after discussion.